packages/java/src/browser/java-client-contribution.ts

```diff
@@ -44,9 +50,28 @@ export class JavaClientContribution extends BaseLanguageClientContribution {
 
     protected onReady(languageClient: ILanguageClient): void {
         languageClient.onNotification(ActionableNotification.type, this.showActionableMessage.bind(this));
+        languageClient.onNotification(StatusNotification.type, this.showStatusMessage.bind(this));
         super.onReady(languageClient);
     }
 
+    protected showStatusMessage(message: StatusReport) {
+        const widget = this.editorManager.currentEditor;
+        if (widget) {
```

Comment:
can you explain why?

Reply:
It probably isn't needed in hindsight, I was just working off of https://github.com/theia-ide/theia/blob/master/packages/editor/src/browser/editor-contribution.ts. I'll remove it